modules/transactions/transactions.js

```diff
@@ -1,4 +1,4 @@
-const confFile = __dirname + '/../../config.json';
+const confFile = '/tmp/tezster/config.json';
```

Comment:
move this to a common config file as well. 
So that we have to change a single place for any change.